### ISSUE TITLE
Show line numbers

### ISSUE THREAD

**Issue by liangkarl** [MaximilianGaedig/tmux-filter#1]
Hi,

Thank you for this great tool!
Is it possible to show the line numbers?
As I'd like to know the line number in the result and jump back to the original pane for looking around what's happenning there.

**Comment by MaximilianGaedig** [comment]
Yep, this could be added, will take a look if I have some spare time. Ideally we'd have a shortcut to let you jump back to the original pane.
